src/PrestaShopBundle/Controller/Admin/MultistoreController.php

```diff
@@ -31,13 +31,14 @@
 use Doctrine\ORM\EntityManager;
 use PrestaShop\PrestaShop\Adapter\Feature\MultistoreFeature;
 use PrestaShop\PrestaShop\Adapter\Shop\Context;
+use PrestaShop\PrestaShop\Core\Domain\Configuration\ShopConfigurationInterface;
+use PrestaShop\PrestaShop\Core\Domain\Shop\ValueObject\ShopConstraint;
 use PrestaShopBundle\Entity\Shop;
 use PrestaShopBundle\Entity\ShopGroup;
 use Symfony\Component\HttpFoundation\Response;
 
 /**
  * This class is responsible for preparing multistore elements that will be displayed in the BO
```

Comment:
```suggestion
 * This class is responsible for preparing multistore elements that will be displayed in the BO
 * It does not control or render a BO page, the items being output are used inside other BO pages.
```

Suggestion, I am wondering if it's useful